sentry-log/src/lib.rs

```diff
@@ -8,32 +8,45 @@
 //! # Examples
 //!
 //! ```
+//! let mut log_builder = pretty_env_logger::formatted_builder();
+//! log_builder.parse_filters("info");
+//! let logger = sentry_log::SentryLogger::with_dest(log_builder.build());
+//!
+//! log::set_boxed_logger(Box::new(logger))
+//!     .map(|()| log::set_max_level(log::LevelFilter::Info))
+//!     .unwrap();
```

Comment:
Can we simplify this?


```suggestion
//! log::set_boxed_logger(Box::new(logger)).unwrap();
//! log::set_max_level(log::LevelFilter::Info);
```